src/package.coffee

```diff
@@ -124,7 +124,10 @@ class Package
             else
               @activateNow()
           catch error
-            @handleError("Failed to activate the #{@name} package", error)
+            if @testMode
```

Comment:
Sorry I was mistaken earlier, but `@testMode` can be replaced with `atom.inSpecMode()`.